modules/machines-edge-machine-pool.adoc

```diff
@@ -0,0 +1,103 @@
+
+:_content-type: CONCEPT
+[id="machines-edge-machine-pool_{context}"]
+= The edge machine pool
+
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zone locations. However, due to the image type and size limitations on Local Zone resources, the default instance type that is created can vary from the traditional worker pool.
+
+The default elastic block store (EBS) for Local Zone locations is `gp2`, which differs from the default worker pool. The preferred list of instance types for the edge compute pool follows the same order of worker pools. Depending on the availability of the location, the system chooses one of those instances. The preferred list of instance types for the edge compute pool includes:
+
+* `m6i.xlarge`
+* `m5.xlarge`
+* `c5d.2xlarge`
+
+The edge compute pool creates new labels that developers can use to deploy their applications onto those locations. The new labels are:
+
+* `node-role.kubernetes.io/edge=''`
+* `zone_type=local-zone`
+* `zone_group=<LOCAL_ZONE_GROUP>`
```

Comment:
Those labels should be updated to:

- `machine.openshift.io/zone-group=<$ZONE_GROUP_NAME>`
- `machine.openshift.io/zone-type=local-zone`

Reply:
Could you please make sure those labels are set correctly on the entire Local Zone book? It's defined on the Enhancement Proposal and installer implementation for phase-1, but it seems to passed from the phase-0 docs review.

Reply:
Beyond what this PR touches, I'm not seeing these labels anywhere.